packages/shared/styles/components/SfHeader.scss

```diff
@@ -1,52 +1,68 @@
-@import '../variables';
+@import "../variables";
 
 $header-mobile-background: $c-light !default;
-$header-mobile-height: 50px !default;
-$header-mobile-justify-content: space-between !default;
+$header-desktop-background: $c-white !default;
+$header-mobile-height: 3.125rem !default;
+$header-desktop-height: 5rem !default;
+$header-desktop-padding: 1.875rem !default;
+
```

Comment:
Check all variables and remove the ones that are not used anywhere.

Reply:
https://github.com/DivanteLtd/storefront-ui/pull/450/commits/1c24542cde6d65197dc7487045d8ef6896e19b09